src/components/Person/Person.jsx

```diff
@@ -1 +1,41 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = () => {
+    if (sex === 'm') {
+      return 'wife';
+    }
+
+    return 'husband';
+  };
```

Comment:
let's make it without a function, but as a ternary

Reply:
Done!